src/ItemFieldSet.tsx

```diff
@@ -46,6 +48,10 @@ interface IItemFieldSetState {
 export class ItemFieldSet extends React.Component<IItemFieldSetProps, IItemFieldSetState> {
   public state = {items: this.props.items};
   public render() {
+    const redirectToNames = this.props.items === INITIAL_STATE.items;
+    if (redirectToNames) {

```

Comment:
Why don't you leave the check inside the if? Much more readable

Reply:
I think I originally had JSX in the redirectToNames variable and then I changed how it worked, forgetting to remove the variable.  Thanks!